src/Raven.Server/ServerWide/Commands/UpdateDatabaseCommand.cs

```diff
@@ -19,7 +21,9 @@ protected UpdateDatabaseCommand(string databaseName, string uniqueRequestId) : b
             DatabaseName = databaseName;
         }
 
-        public abstract string UpdateDatabaseRecord(DatabaseRecord record, long etag);
+        public abstract void UpdateDatabaseRecord(DatabaseRecord record, long etag);
+        
+        public virtual void CleanupLeftovers(ClusterOperationContext ctx, Table items,  Logger clusterAuditLog){}
```

Comment:
I would prefer a more generic name like `AfterDatabaseRecordUpdate` or sth like that. Maybe it would be better just to change the name of the `UpdateDatabaseRecord` and handle everything there? I do not like the idea of passing info from one method to another via private fields.

Also can we fix formatting?

Reply:
We need to separate because the `UpdateDatabaseRecord` may fail (consistently) across nodes.